Traceback (most recent call last):
  File "run_mazes.py", line 127, in <module>
    path_gif = path_gif, path_sampled_maze=path_sampled_maze, path_metrics=path_metrics, path_maze_visu=path_maze_visu, gpu_name=gpu_name)
  File "/home/pierre/Git/teachDeepRL/teachDRL/spinup/algos/ppo/ppo.py", line 212, in ppo
    pi, logp, logp_pi, v = actor_critic(x_ph, a_ph, **ac_kwargs)
  File "/home/pierre/Git/teachDeepRL/teachDRL/spinup/algos/ppo/core.py", line 140, in convolutional_actor_critic
    with tf.variable_scope('pi'):
AttributeError: module 'tensorflow' has no attribute 'variable_scope'